templates/severity_high_performance_impact_needinfo.txt

```diff
@@ -0,0 +1,4 @@
+The severity field for this bug is set to {{ extra[bugid]["severity"] }}. However, the `Performance Impact` field flags this bug as having a high impact on the performance.
+:{{ nickname }}, could you consider increasing the severity of this performance-impacting bug?
```

Comment:
```suggestion
:{{ nickname }}, could you consider increasing the severity of this performance-impacting bug?
```
Maybe we could say something like "Or, if you think the impact is lower than previously thought, could you request retriage from the performance team?"

Reply:
I like @marco-c's suggestion about requesting retriage.

Reply:
Done in 588a279b0b993d40548e76ad937324ad2ffc5f89